docs/tasks.yaml

```
@@ -28,10 +28,10 @@ tasks:
     dir: "{{.TASKFILE_DIR}}"
     deps:
       - ":init"
-      - task: ":utils:validate-checksum"
+      - task: ":utils:checksum:validate"
         vars:
           CHECKSUM_FILE: "{{.CHECKSUM_FILE}}"
-          DATA_DIR: "{{.OUTPUT_DIR}}"
+          INCLUDE_PATTERNS: ["{{.OUTPUT_DIR}}/**/*"]
```

```suggestion
          INCLUDE_PATTERNS: ["{{.OUTPUT_DIR}}/**/*"]
```

Why do we need the `/**/*`?